apps/builder/app/shared/instance-utils.test.tsx

```diff
@@ -3165,6 +3171,101 @@ describe("insert webstudio fragment copy", () => {
       availableDataSources: new Set(),
     });
     expect(data.instances.size).toEqual(4);
-    expect(newInstanceIds.size).toEqual(4);
+    expect(newInstanceIds.size).toEqual(5);
+  });
+});
+
+describe("support copy paste of :root styles", () => {
+  const css = (strings: TemplateStringsArray, ...values: string[]) => {
```

Comment:
this test looks quite complex for a test, looks like we are missing some abstraction

Reply:
this is an abstraction around the logic I need to test.
Maybe it will go to module level in the future, didn't want to spend time on refactoring

Reply:
I think we have now 2 very similar functions, another one i wrote in plugin-webflow.test.tsx called toCss and inlined it in the file with the same reasoning as you now

Reply:
problem with these is that next time we will create another one and another one, because we don't know whats inside all individual modules unless its used everywhere

Reply:
Anyway, test is simpler with this. Will refactor the rest of tests some time later. Maybe find the right abstraction. But not in context of this PR.